### PR TITLE
Monitor class documentation enhancement

### PR DESCRIPTION
**Pull request by wilove997**

### CODE REVIEW

File: pyaedt/modules/monitor_icepak.py [hunk 1]
```diff
@@ -619,30 +630,31 @@ def properties(self):
...
     def delete(self):
         """
-        Delete a monitor object
+        Delete a monitor object.
 
         Returns
         -------
-        True if successful
+        bool
+            True if successful.
```
Comment (MaxJPRey) [inline pyaedt/modules/monitor_icepak.py:638]:
```suggestion
            ``True`` if successful.
```

File: pyaedt/modules/monitor_icepak.py [hunk 2]
```diff
@@ -510,20 +518,21 @@ def _delete_removed_monitors(self):
     @pyaedt_function_handler()
     def insert_monitor_object_from_dict(self, monitor_dict, mode=0):
         """
-        Get the object that the monitor is applied to
+        Get the object that the monitor is applied to.
```
Comment (MaxJPRey) [inline pyaedt/modules/monitor_icepak.py:521]:
```suggestion
        Insert a monitor.
```

File: pyaedt/generic/configurations.py [hunk 3]
```diff
@@ -1470,6 +1470,23 @@ def _export_mesh_operations(self, dict_out):
...
+        m_object : lost or str or int
+            Name or id (or list of these) of the geometry object being monitored.
+        m_quantity : list or str
+            Name or list of names of the quantity being monitored.
+        m_name : str
+            Name of the monitor object.
+        Returns
+        -------
+        bool
+            ``True`` if successful
```
Comment (MaxJPRey) [inline pyaedt/generic/configurations.py:1488]:
```suggestion
            ``True`` if successful.
```

File: pyaedt/generic/configurations.py [hunk 4]
```diff
@@ -1470,6 +1470,23 @@ def _export_mesh_operations(self, dict_out):
 
     @pyaedt_function_handler()
     def update_monitor(self, m_case, m_object, m_quantity, m_name):
+        """
```
Comment (MaxJPRey) [inline pyaedt/generic/configurations.py:1473]: Here the description is missing.

**Review by MaxJPRey — APPROVED** [review]
Nice improvements @wilove997 .

**Comment by wilove997** [comment]
Thanks, and thanks for the suggestions @MaxJPRey 👍

**Pull request merged**